### PR TITLE
Frontpage in angularJS with bootstrap, angular routing.

### PR DESCRIPTION
**Pull request by PatrykDawczak**

### CODE REVIEW

**Comment by TomaszGabrysiak** [comment]
First of all, the commit message is not eloquent. Please use more informative messages

File: app/index-async.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
...unction(){e[m]&&!/^c|loade/.test(e[m])||f||(e.onload=e[o]=null,f=1,c())},e.async=1,e.src=a,d.insertBefore(e,d.firstChild)}function q(a,b){p(a,function(a){return!b(a)})}var d=b.getElementsByTagName("head")[0],e={},f={},g={},h={},i="string",j=!1,k="push",l="DOMContentLoaded",m="readyState",n="addEventListener",o="onreadystatechange",p=function(a,b){for(var c=0,d=a.length;c<d;++c)if(!b(a[c]))return j;return 1};!b[m]&&b[n]&&(b[n](l,function r(){b.removeEventListener(l,r,j),b[m]="complete"},j),b[m]="loading");var s=function(a,b,d){function o(){if(!--m){e[l]=1,j&&j();for(var a in g)p(a.split("|"),n)&&!q(g[a],n)&&(g[a]=[])}}function n(a){return a.call?a():e[a]}a=a[k]?a:[a];var i=b&&b.call,j=i?b:d,l=i?a.join(""):b,m=a.length;c(function(){q(a,function(a){h[a]?(l&&(f[l]=1),o()):(h[a]=1,l&&(f[l]=1),t(s.path?s.path+a+".js":a,o))})},0);return s};s.get=t,s.ready=function(a,b,c){a=a[k]?a:[a];var d=[];!q(a,function(a){e[a]||d[k](a)})&&p(a,function(a){return e[a]})?b():!function(a){g[a]=g[a]||[],g[a][k](b),c&&c(d)}(a.join("|"));return s};var u=a.$script;s.noConflict=function(){a.$script=u;return this},typeof module!="undefined"&&module.exports?module.exports=s:a.$script=s}(this,document,setTimeout)
```
Comment (TomaszGabrysiak) [inline app/index-async.html]: what is this?

File: app/index.html [hunk 2]
```diff
@@ -1,65 +1,34 @@
...
-            <ul class="nav navbar-nav">
-                <li class="active"><a href="#slide1">Home</a></li>
-                <li><a href="#slide2">Napoje</a></li>
-                <li><a href="#slide3">Przystawki</a></li>
-                <li><a href="#slide4">Danie główne</a></li>
-                <li><a href="#slide5">Zamówienia</a></li>
-            </ul>
-        </div>
+  <div class="container">
+    <div class="masthead">
```
Comment (TomaszGabrysiak) [inline app/index.html]: I would rather have only bootstrap's classes

File: app/index.html [hunk 3]
```diff
@@ -1,65 +1,34 @@
...
+        <ul class="menu">
```
Comment (TomaszGabrysiak) [inline app/index.html]: you can use class .list-inline to achieve exactly the same behaviour

File: app/view1/view1.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+angular.module('myApp.view1', ['ngRoute'])
+
+.config(['$routeProvider', function($routeProvider) {
+  $routeProvider.when('/view1', {
```
Comment (TomaszGabrysiak) [inline app/view1/view1.js]: I would rather use ui.router but it's up to you

**Comment by TomaszGabrysiak** [comment]
OK, I wrote couple comments. Looks not so bad

File: app/index.html (hunk 2 above)
Comment (PatrykDawczak) [inline app/index.html]: I don't know why I use two div's, so I'm delete "masthead"

File: app/index.html (hunk 3 above)
Comment (PatrykDawczak) [inline app/index.html]: Done

File: app/view1/view1.js (hunk 4 above)
Comment (PatrykDawczak) [inline app/view1/view1.js]: I'm try add ui.router.

**Pull request merged**